.github/workflows/release.yaml

```
@@ -93,7 +93,7 @@ jobs:
         uses: actions/checkout@v3
       - name: Move Snap to project root directory
         run: cp -r packages/snap/ .
-      - name: 📦 Build Snap
+      - name: build
```

The name does not need to be modified, perhaps the `id: build ` needs to be set.

```suggestion
      - name: 📦 Build Snap
        id: build
```